src/main/java/com/tstepnik/planner/domain/user/userregistrationdto/UserRegisterDTO.java

```diff
@@ -0,0 +1,78 @@
+package com.tstepnik.planner.domain.user.userregistrationdto;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Size;
+
+public class UserRegisterDTO {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotEmpty
+    @Size(min=3)
+    private String login;
+
+    private String firstName;
+
+    private String lastName;
+
+    @NotEmpty
+    @Size(min=3)
+    private String email;
```

Comment:
missing email annotation

Reply:
don't rush development. Check all entities and dtos while making changes in other classes that also might apply here.

Reply:
Now I'm little scarry do any PR because always i can miss sth :D